omnipaxos/src/sequence_paxos/follower.rs

```diff
@@ -167,26 +169,19 @@ where
                         ],
                         "storage error while trying to write log entries",
                     );
-                    Accepted {
-                        n: accsync.n,
-                        accepted_idx: accepted_idx
-                            .expect("storage error while trying to write log entries"),
-                    }
                 }
             };
-            match accsync.stopsign {
-                Some(ss) => {
-                    if let Some(ss_entry) = self.internal_storage.get_stopsign() {
-                        if !ss_entry.decided(old_decided_idx) {
-                            self.accept_stopsign(ss);
-                        }
-                    } else {
-                        self.accept_stopsign(ss);
-                    }
-                    accepted.accepted_idx = self.internal_storage.get_accepted_idx();
-                }
-                None => self.forward_pending_proposals(),
+            if accsync.stopsign.is_none() {
+                self.forward_pending_proposals();
+                //TODO: foward pending stopsign like in handle_majority_promises?
```

Comment:
no need to forward StopSign here. In handle_majority_promises we need to adopt the StopSign because it might be chosen. Here it is not chosen so don't need to do it.